client/src/ts/can_msg_defs.ts

```diff
@@ -0,0 +1,54 @@
+// For setting the CAN device
+export const enum CanDevice {
```

Comment:
I'm assuming this only exists as a hack?

Reply:
Yeah.